### PR TITLE
Feature/add uri healthchecks

### PR DESCRIPTION
**Pull request by Noviny**

### CODE REVIEW

**Comment by Noviny** [comment]
@bassjacob This PR is now ready for review

File: src/healthchecks/canQueryUriFactory.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const canQueryUri = require('./canQueryUri');
```
Comment (bassjacob) [inline src/healthchecks/canQueryUriFactory.js]: Semantically, names of classes should be PascalCase

File: src/index.js [hunk 2]
```diff
@@ -4,5 +4,7 @@ module.exports = {
 	healthchecks: {
 		CanQueryList: require('./healthchecks/CanQueryList'),
 		canQueryListFactory: require('./healthchecks/canQueryListFactory'),
+		CanQueryUri: require('./healthchecks/CanQueryUri'),
```
Comment (bassjacob) [inline src/index.js:7]: this will fail on any OS that isn't MacOS. The case of the filename is different to the case of the string?

**Review by bassjacob — CHANGES_REQUESTED** [review]
case related changes

**Comment by Noviny** [comment]
Fix'd @bassjacob

**Review by bassjacob — APPROVED** [review]

**Pull request merged**